File: backend/internal/constants/phase_decision.go
package constants

type PhaseDecision string

const (
	PhaseDecisionAccepted   PhaseDecision = "accepted"
	PhaseDecisionFollowUp   PhaseDecision = "follow_up"
	PhaseDecisionFalseAlarm PhaseDecision = "false_alarm"
)

func (d PhaseDecision) Valid() bool {
	switch d {
	case PhaseDecisionAccepted, PhaseDecisionFollowUp, PhaseDecisionFalseAlarm:
		return true
	default:
		return false
	}
}

func PhaseDecisionValues() []string {
	return []string{
		string(PhaseDecisionAccepted), string(PhaseDecisionFollowUp), string(PhaseDecisionFalseAlarm),
	}
}

File: frontend/src/types/enums/phase-decision.ts
export const phaseDecisions = ['accepted', 'follow_up', 'false_alarm'] as const
export type PhaseDecision = typeof phaseDecisions[number]

export const phaseDecisionLabels: Record<PhaseDecision, string> = {
  accepted: '认可证据',
  follow_up: '要求补查',
  false_alarm: '排除误报',
}
